src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
```

Comment:
The naming for `array` is not descriptive. Consider using a more descriptive name like `stateHistory` or `statesSnapshot` to reflect the purpose of the array.